deployment/terraform/dump.go

```diff
@@ -257,12 +252,36 @@ func (t *Terraform) executeDatabaseCommands(extraCommands []deployment.Cmd) erro
 func (t *Terraform) executeCommands(commands []deployment.Cmd) error {
 	for _, c := range commands {
 		mlog.Info(c.Msg)
+
+		errors := make(chan error, len(c.Clients))
+		wg := sync.WaitGroup{}
+
 		for _, client := range c.Clients {
-			mlog.Debug("Running cmd", mlog.String("cmd", c.Value))
-			if out, err := client.RunCommand(c.Value); err != nil {
-				return fmt.Errorf("failed to run cmd %q: %w %s", c.Value, err, out)
-			}
+			wg.Add(1)
+			go func() {
+				defer wg.Done()
+				mlog.Debug("Running cmd", mlog.String("cmd", c.Value))
+				if out, err := client.RunCommand(c.Value); err != nil {
+					errors <- fmt.Errorf("failed to run cmd %q: %w %s", c.Value, err, out)
```

Comment:
Pass the IP of the instance, otherwise it will be difficult to debug on which machine it failed.

Reply:
Please check: https://github.com/mattermost/mattermost-load-test-ng/pull/863/commits/4517d3859521acd1976b05ba632358fd8639d032

I prefer the hostname but that requires more changes.